### PR TITLE
Manual backport: (Issue: #20228, PR: #20954, #21051) Fix sometimes table doesn't resize

### LINKED ISSUE #20228: When switching from map visualization to base table, table doesn't take up full screen
**Describe the bug**
When you are viewing a map visualization, and you click the "raw data" toggle to view the base data, the table doesn't take up the full horizontal width of the screen.

**To Reproduce**
1. Go to Sample Database -> People -> Visualization -> Map
2. Click "Done" at the bottom of the viz settings sidebar (this is important to repro!)
3. Click the toggle at the bottom of the page to view the raw data

**Expected behavior**
Should take up full width of page

**Screenshots**

https://user-images.githubusercontent.com/32746338/152409858-ca830095-f806-462a-86ed-7ea242cb76eb.mov

**Information about your Metabase Installation:**
Current master

**Severity**
Cosmetic

### PR DESCRIPTION
**Pull request by WiNloSt**
Manual backport of PR #20954, #21051 to fix Issue #20228

### CODE REVIEW

**Review by flamber — APPROVED** [review]
Good to merge, when #21051 is added too

**Comment by WiNloSt** [comment]
> Good to merge, when #21051 is added too

Right... Why didn't I think of that 🤦 . I thought I want to merge this PR before creating another backport PR.

File: frontend/src/metabase/components/DebouncedFrame.jsx [hunk 1]
```diff
@@ -32,28 +32,34 @@ export default class DebouncedFrame extends React.Component {
...
     }
     if (
-      this.props.width !== nextProps.width ||
-      this.props.height !== nextProps.height
+      this.state.width !== nextProps.width ||
+      this.state.height !== nextProps.height
     ) {
       if (this.state.width == null || this.state.height == null) {
-        this.setSize(nextProps.width, nextProps.height);
+        this.updateSizeDebounced();
```
Comment (flamber) [inline frontend/src/metabase/components/DebouncedFrame.jsx:58]: Shouldn't it be `this.setSizeDebounced(nextProps.width, nextProps.height);` ?
Merge conflict or am I misreading the two PRs?

Comment (WiNloSt) [reply frontend/src/metabase/components/DebouncedFrame.jsx:58]: @flamber The first PR [change the line to](https://github.com/metabase/metabase/pull/20954/files#diff-cfcbb156d18ef93cd9c3a11763ba671ebe6c937102ecd10412113b745362eb7aR52) `this.setSizeDebounced(nextProps.width, nextProps.height);` then the [second PR which fixes the regression change it to](https://github.com/metabase/metabase/pull/21051/files#diff-cfcbb156d18ef93cd9c3a11763ba671ebe6c937102ecd10412113b745362eb7aR58) `this.updateSizeDebounced()`.

This is already correct.

Comment (flamber) [reply frontend/src/metabase/components/DebouncedFrame.jsx:58]: Sorry, got lost in the commits

**Pull request merged**